sdk/playwrighttesting/microsoft-playwright-testing/src/utils/utils.ts

```diff
@@ -143,6 +143,12 @@ export const getPlaywrightVersion = (): string => {
   return process.env[InternalEnvironmentVariables.MPT_PLAYWRIGHT_VERSION]!;
 };
 
+export const getReporterVersion = (): string => {
+  // eslint-disable-next-line @typescript-eslint/no-require-imports
+  const version = require("../../package.json").version;
```

Comment:
Is there a way we can move away from using `require` so that we can use ESM as our core soon?

Reply:
we are soon planning to move this package to ESM/vitest. Will take up this change along with that.